tests/helper/helper_generic.go

```diff
@@ -319,6 +319,12 @@ func JsonPathDoesNotExist(json string, path string) {
 		fmt.Sprintf("content should not contain %q but is %q", path, result.String()))
 }
 
+// JsonPathExist expects that the content of the path exists in the JSON string
+func JsonPathExist(json string, path string) {
+	result := gjson.Get(json, path)
+	Expect(result.Exists()).To(BeTrue(),
+		fmt.Sprintf("content should not contain %q but is %q", path, result.String()))
```

Comment:
The error message could be a bit misleading, as we are asserting that the JSON path does exist. So I'd suggest:
```suggestion
		fmt.Sprintf("content should contain %q", path))
```

Otherwise, I got:
```
[FAILED] content should not contain "0.ports" but is ""
```